CommonTools/UtilAlgos/interface/ObjectSelector.h

```diff
@@ -17,6 +17,7 @@
 #include "FWCore/Framework/interface/ConsumesCollector.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 #include "FWCore/Utilities/interface/InputTag.h"
+#include "CommonTools/CandAlgos/interface/CandDecaySelector.h"'
```

Comment:
I don't understand why CandDecaySelector is needed here
instead of the headers where this input is actually needed